src/common/hooks/useBoundPieces/useBoundPieces.js

```diff
@@ -15,7 +15,7 @@ const useBoundPieces = (itemId, options = {}) => {
   const { enabled = true, ...otherOptions } = options;
   const ky = useOkapiKy();
   const [namespace] = useNamespace({ key: 'bound-pieces' });
-  const filterQuery = `itemId==${itemId} and isBound==true`;
+  const filterQuery = `bindItemId==${itemId} and isBound==true`;
```

Comment:
adding appropriate index for bindItemId in scope of: https://github.com/folio-org/mod-orders-storage/pull/427